### DISCUSSION TITLE
Min and max price filter

### DISCUSSION THREAD

**Discussion by twin-elements** [coreshop/CoreShop#2765] [Q&A]
Need to create a filter for MIN and MAX price.
I don't understand how to index the prices so that I can use the RANGE filter. 

Which data exactly should I index (storeValues?) and with which configuration?

![image](https://github.com/user-attachments/assets/de8e7ff3-4c87-4ece-bc17-ab5fc7658e4b)

The Coreshop demo is just missing an example of filters. This would really help when working with Coreshop.

Thank you in advance for your help

**Comment by dpfaffenbauer** [comment]
@twin-elements Create a custom interpreter and fetch the prices you need, be careful though, Price Rules are not indexed! So only the prices in the store values can be indexed.

**Reply by twin-elements** [reply]
I understand that first I have to create Index Extension and in it I have to add the columns: price_{store_id} - price_1 (eur), price_2 (pln), price_3(usd) to the index table?

**Reply by dpfaffenbauer** [reply]
does not need to be an extension, can be just index fields.

**Reply by twin-elements** [reply]
Please clarify some points for me very much.

1. in the indexed fields settings, I add storeValues
![image](https://github.com/user-attachments/assets/1d078775-2d51-4f95-9f2d-b7e7448d9062)

2. A storeValues column has been created in the mysql table
![image](https://github.com/user-attachments/assets/fb18ecb6-ccc5-4075-a440-e36f732b64ff)

I have several stores and for each store the price will be different (PLN, EUR, USD). 
How do I store the prices in the storeValues field? I can't store several values in one field because the field must be an integer for a filter with a range of prices.
Don't I need to add additional columns for each price?

**Reply by twin-elements** [reply]
@dpfaffenbauer I will be incredibly grateful to you for your help Dominik.

**Reply by dpfaffenbauer** [reply]
@twin-elements Sorry, I somehow missed your reply. Yes, you need either separate indices or separate columns for each currency. CoreShop doesn't support that out of the box automatically
